### PR TITLE
Fixing wow versioning things

### PR DESCRIPTION
**Pull request by cwilson21**
In a small patch today, Blizzard fixed how the version is reported for wrath classic. Adding these changes here as the addon breaks now when clients get the new update.

### CODE REVIEW

**Comment by cwilson21** [comment]
So to summarize, blizzard hotfixed the game version (from TBC to WoTLK) and it broke a bunch of addons that do version checking for config.  I was able to fix this by opening the entire addons folder in an IDE so you can global find/replace the test string.
Search for: WOW_PROJECT_BURNING_CRUSADE_CLASSIC
Replace with:  WOW_PROJECT_WRATH_CLASSIC

Instead of just replacing I added an additional if so that it will use the wowhead wrath URL.

**Comment by evil-morfar** [comment]
Thanks for your help!

File: .luacheckrc [hunk 1]
```diff
@@ -19416,6 +19416,7 @@ globals = {
 	"WOW_MOUSE_NOT_FOUND",
 	"WOW_PROJECT_CLASSIC",
 	"WOW_PROJECT_BURNING_CRUSADE_CLASSIC",
+	"WOW_PROJECT_BURNING_CRUSADE_CLASSIC",
```
Comment (D-Joke) [inline .luacheckrc:19419]: @cwilson21 @evil-morfar This has to be "WOW_PROJECT_WRATH_CLASSIC" or am i wrong?

Comment (cwilson21) [reply .luacheckrc:19419]: Yes it does my bad I did not see an issue when i was testing it in my local game.

Comment (evil-morfar) [reply .luacheckrc:19419]: I did see it before merging, but since its only a definition table it doesn't really matter :)

Comment (cwilson21) [reply .luacheckrc:19419]: Let me know if you need any help getting things fully updated for wrath, more than happy to help where I can. Not a professional with LUA but I can stumble my way through things.

**Pull request merged**